presto-parser/src/main/antlr4/com/facebook/presto/sql/parser/SqlBase.g4

```diff
@@ -273,6 +272,7 @@ primaryExpression
     | identifier '->' expression                                                     #lambda
     | '(' identifier (',' identifier)* ')' '->' expression                           #lambda
     | '(' query ')'                                                                  #subqueryExpression
+    | EXISTS '(' query ')'                                                           #exists
```

Comment:
According to the spec, that's not a valid expression. `EXISTS` is a `<predicate>`, which is a form of `<boolean primary>`, which can only be operated on with `AND`, `OR`, `NOT`, `IS/IS NOT TRUE/FALSE/UNKNOWN`.

We may want to extend Presto to support it, but we need to consider the implications (possibly grammar ambiguities, etc)